src/platform/landing-pages/dev-template.ejs

```diff
@@ -0,0 +1,168 @@
+<html>
```

Comment:
This whole template is essentially [`src/site/layouts/page-react.html`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/layouts/page-react.html) with a few things stripped out since it will only be used for local development. (Goodbye, IE-specific ~garbage~ comment directive thingies!)